python/test/test_binding.py

```diff
@@ -0,0 +1,42 @@
+from ionpy import Node, Builder, Buffer, PortMap, Port, Param, Type, TypeCode
+import numpy as np # TODO: rewrite with pure python
+
+def test_binding():
```

Comment:
Can you replicate the C++ [testcase](https://github.com/fixstars/ion-kit/blob/master/test/port-binding.cc) for port binding using Python API? I want to keep the same C++ and Python API semantics.